pkg/proxy/apis/kubeproxyconfig/types.go

```diff
@@ -168,6 +177,12 @@ const (
 	ProxyModeKernelspace ProxyMode = "kernelspace"
 )
 
+type InterfaceType string
+
+const (
+	DefaultRoute InterfaceType = "default-route"
```

Comment:
I am still not convinced we have a use-case for this.  What if we didn't have this?  Can't a CIDR always fill in for this?

Reply:
I can not find a user case, sadly. 

Let's remove this one.